util/process_triggers_folder.ts

```diff
@@ -159,51 +179,40 @@ const processFile = async (filename: string) => {
     const options = { ...defaultRaidbossOptions };
     UserConfig.evalUserFile(contents, options);
   } catch (e) {
-    console.error(`${filename}: Failed eval.`);
+    console.error(`${distFilePath}: Failed eval.`);
     console.log(e);
     process.exit(5);
   }
 
   // Overwrite the file.
-  fs.writeFileSync(filename, contents);
+  fs.mkdirSync(path.dirname(distFilePath), { recursive: true });
+  fs.writeFileSync(distFilePath.replace('.ts', '.js'), contents);
 
   // Copy timeline file if present
-  const importPath = path.join(
-    '..',
-    originalFilename.replace(path.extname(originalFilename), '.ts'),
+  const importPath = ('../ui/raidboss/data/' + path.relative(root, originalFilename)).replace(
```

Comment:
Thanks for fixing this.  I have no idea how this ever worked locally without the slash replacement.

Reply:
yes, it didn't work...